python/pyarrow/array.pxi

```diff
@@ -2746,6 +2762,29 @@ cdef class StructArray(Array):
         result.validate()
         return result
 
+    def sort(self, order="ascending", fieldname=None):
+        """
+        Sort the StructArray
+
+        Parameters
+        ----------
+        order : "ascending" or "descending"
+            The order of the sorting.
```

Comment:
Maybe can use the same entry as for array_sort_indices to keep things consistent:

```
order : str, default "ascending"
    Which order to sort values in.
    Accepted values are "ascending", "descending".
```

Reply:
Changed as suggested